### ISSUE TITLE
Feature Suggestion: Force stopping a jail

### ISSUE THREAD

**Issue by vendion** [iocage/iocage#532]
If would be nice to be able to force stop a jail with `iocage` rather than having to fall back to `jail -r <JID>` to do so. Some reasons why this may be helpful could be the jail stops responding or something goes majorly wrong and you need to rollback no a snapshot.

A very terrible example might by
```
$ sudo iocage create -n test
$ sudo iocage start test
$ sudo iocage snapshot test
$ sudo iocage console test
# rm -rf /* // inside the test jail
(new terminal/window/whatever)
$ sudo iocage stop test // rollback requires jail to be stopped
* Stopping test
  + Running prestop OK
Traceback (most recent call last):
  File "/usr/local/bin/iocage", line 10, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/iocage/cli/stop.py", line 54, in cli
    ioc.IOCage(exit_on_error=True, jail=jail, rc=rc).stop()
  File "/usr/local/lib/python3.6/site-packages/iocage/lib/iocage.py", line 1655, in stop
    exit_on_error=self.exit_on_error)
  File "/usr/local/lib/python3.6/site-packages/iocage/lib/ioc_stop.py", line 52, in __init__
    self.__stop_jail__()
  File "/usr/local/lib/python3.6/site-packages/iocage/lib/ioc_stop.py", line 142, in __stop_jail__
    stdout=f, stderr=su.PIPE)
  File "/usr/local/lib/python3.6/subprocess.py", line 291, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['setfib', '0', 'jexec', 'ioc-test', '/bin/sh', '/etc/rc.shutdown']' returned non-zero exit status 1.
$ sudo jail -r 7 // 7  is JID according to iocage list
$ sudo iocage rollback -n <snapshot name> test
$ sudo iocage start test
```

In this case the Python interpreter crashing, or at the very least erroring out, is okay as neither `/bin/sh` or `/etc/rc.shutdown` existed in the jail at that point.

**Comment by Avatat** [comment]
@skarekrow, could I implement this?

**Comment by skarekrow** [comment]
@Avatat Absolutely be my guest!

**Comment by jsegaert** [comment]
@skarekrow   This feature was implemented as per #553 so this issue can be closed.
